src/process/task.rs

```diff
@@ -188,4 +206,30 @@ mod tests {
         assert!(found_one);
         assert!(found_two);
     }
+
+    #[test]
+    fn test_task_children() {
+        // Use tail -f /dev/null to create two infinite processes
+        let mut command = process::Command::new("tail");
+        command.arg("-f").arg("/dev/null");
+        let (mut child1, mut child2) = (command.spawn().unwrap(), command.spawn().unwrap());
+
+        // Cargo tests run on multiple threads under the same process by default
+        // There doesn't seem to be an easy way to get the current thread ID
```

Comment:
What about https://docs.rs/rustix/latest/rustix/thread/fn.gettid.html to get the thread ID?

Reply:
Good point. Didn't realize rustix has that utility. I'll change it.